### PR TITLE
applied camelcasing

### PR DESCRIPTION
**Pull request by tarupathak**

### CODE REVIEW

File: src/MyAccount/Main/MainApp.tsx [hunk 1]
```diff
@@ -102,7 +102,7 @@ const Account = () => {
       });
   }
   const inputphoto = () => {
```
Comment (Tech-Shreyansh) [inline src/MyAccount/Main/MainApp.tsx]: convert inputphoto to camelcase

File: src/homepage/createpost.tsx [hunk 2]
```diff
@@ -32,12 +32,12 @@ function Post(props: any) {
   const inputphoto = () => {
     setImageData([]);
     imgarray = [];
-    document.getElementById("inpphoto")?.click();
+    document.getElementById("inpPhoto")?.click();
   };
   const inputvideo = () => {
```
Comment (Tech-Shreyansh) [inline src/homepage/createpost.tsx]: convert inputvideo to camelcase

**Comment by Tech-Shreyansh** [comment]
Please go through function names as well

**Comment by tarupathak** [comment]
> Please go through function names as well

Okay Sure. Thanks

**Review by Tech-Shreyansh — APPROVED** [review]

**Pull request merged**